### PR TITLE
[enterprise-4.14] OCPBUGS#32522: Doc improvements related to LVMCluster CR

### PR DESCRIPTION
**Pull request by sr1kar99**
Version(s):
4.14

Issue:
[OCPBUGS-32522](https://issues.redhat.com/browse/OCPBUGS-32522)

Link to docs preview:
[Preview](https://74937--ocpdocs-pr.netlify.app/openshift-enterprise/latest/storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms#about-lvmcluster_logical-volume-manager-storage)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by sr1kar99** [comment]
@radeore 
Can you please review this PR?
Thanks!

File: modules/lvms-creating-lvms-cluster-using-cli.adoc [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+lvmcluster/lvmcluster created
+----
+
+.Verification
+
+. Check that the `LVMCluster` CR is in the `Ready` state:
++
+[source, terminal]
+----
+$ oc get lvmclusters.lvm.topolvm.io -o jsonpath='{.items[*].status}' -n <namespace>
```
Comment (radeore) [inline modules/lvms-creating-lvms-cluster-using-cli.adoc:75]:
```suggestion
$ oc get lvmclusters.lvm.topolvm.io -o jsonpath='{.items[*].status.state}' -n <namespace>
```

Comment (sr1kar99) [reply modules/lvms-creating-lvms-cluster-using-cli.adoc:75]: Updated as per the suggestion!

File: modules/lvms-about-adding-devices-to-a-vg.adoc [hunk 2]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-adding-devices-to-a-vg_{context}"]
+= About adding devices to a volume group
+
+The `deviceSelector` field in the `LVMCluster` CR contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
```
Comment (adellape) [inline modules/lvms-about-adding-devices-to-a-vg.adoc:9]:
```suggestion
The `deviceSelector` field in the `LVMCluster` custom resource (CR) contains the configuration to specify the paths to the devices that you want to add to the LVM volume group.
```

File: modules/lvms-about-creating-lvmcluster-cr.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+You can create an `LVMCluster` custom resource (CR) by using the OpenShift CLI (`oc`) or the {product-title} web console. If you have installed {lvms} by using {rh-rhacm-first}, you can also create an `LVMCluster` CR by using {rh-rhacm}. 
+
+Upon creating the `LVMCluster` CR, {lvms} creates the following system-managed CRs:
+
+* A `storageClass` and `volumeSnapshotClass` for each device class.
++
+[NOTE]
+====
+{lvms} configures the name of the storage class and volume snapshot class in the format `lvms-<device_class_name>`, where, `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR. For example, if the `deviceClasses.name` field is set to vg1, the name of the storage class and volume snapshot class is `lvms-vg1`.
```
Comment (adellape) [inline modules/lvms-about-creating-lvmcluster-cr.adoc:17]:
```suggestion
{lvms} configures the name of the storage class and volume snapshot class in the format `lvms-<device_class_name>`, where, `<device_class_name>` is the value of the `deviceClasses.name` field in the `LVMCluster` CR. For example, if the `deviceClasses.name` field is set to `vg1`, the name of the storage class and volume snapshot class is `lvms-vg1`.
```

File: modules/lvms-creating-lvms-cluster-using-cli.adoc [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+----
+status:
+  deviceClassStatuses:
+    - name: vg1
+      nodeStatus:
+        - node: my-node-1.example.com 
+          reason: no available devices found for volume group
+          status: Failed
+  state: Failed
+----
```
Comment (adellape) [inline modules/lvms-creating-lvms-cluster-using-cli.adoc:122]: There's a typo ("failue"), but also I feel like this sentence could just be the title for the block instead?
```suggestion
.Example `status` field with the reason for failure
[source,yaml]
----
status:
  deviceClassStatuses:
    - name: vg1
      nodeStatus:
        - node: my-node-1.example.com 
          reason: no available devices found for volume group
          status: Failed
  state: Failed
----
```

File: modules/lvms-about-lvmcluster-cr.adoc [hunk 5]
```diff
@@ -0,0 +1,141 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="about-lvmcluster_{context}"]
+= About the LVMCluster custom resource
+
+You can configure the `LVMCluster` CR to perform the following actions:
```
Comment (adellape) [inline modules/lvms-about-lvmcluster-cr.adoc:9]:
```suggestion
You can configure the `LVMCluster` custom resource (CR) to perform the following actions:
```

**Comment by adellape** [comment]
@sr1kar99 Can you confirm that this 4.14 PR is related to https://github.com/openshift/openshift-docs/pull/72137? To make it clear during merge review why this is only targeting the `enterprise-4.14` branch. Thank you!

**Comment by sr1kar99** [comment]
> @sr1kar99 Can you confirm that this 4.14 PR is related to #72137? To make it clear during merge review why this is only targeting the `enterprise-4.14` branch. Thank you!

Hi @adellape 
Yes! You are right. Similar PR for 4.15+ has been merged already. The reason for having a separate PR for 4.14 is that 4.14 and lower versions support this Operator only on single-node OpenShift.

Thanks for adding the 4.15+ PR link. :)

File: modules/lvms-about-adding-devices-to-a-vg.adoc (hunk 2 above)
Comment (sr1kar99) [reply modules/lvms-about-adding-devices-to-a-vg.adoc:9]: Updated as per the suggestion!

File: modules/lvms-about-creating-lvmcluster-cr.adoc (hunk 3 above)
Comment (sr1kar99) [reply modules/lvms-about-creating-lvmcluster-cr.adoc:17]: Updated as per the suggestion.

File: modules/lvms-creating-lvms-cluster-using-cli.adoc (hunk 4 above)
Comment (sr1kar99) [reply modules/lvms-creating-lvms-cluster-using-cli.adoc:122]: Updated as per the suggestion.

**Review by sheriff-rh — APPROVED** [review]
Changes look good; merging.

**Pull request merged**